tests/components/letpot/test_config_flow.py

```diff
@@ -0,0 +1,147 @@
+"""Test the LetPot config flow."""
+
+from typing import Any
+from unittest.mock import AsyncMock, patch
+
+from letpot.exceptions import LetPotAuthenticationException, LetPotConnectionException
+import pytest
+
+from homeassistant.components.letpot.const import (
+    CONF_ACCESS_TOKEN_EXPIRES,
+    CONF_REFRESH_TOKEN,
+    CONF_REFRESH_TOKEN_EXPIRES,
+    CONF_USER_ID,
+    DOMAIN,
+)
+from homeassistant.config_entries import SOURCE_USER
+from homeassistant.const import CONF_ACCESS_TOKEN, CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from . import AUTHENTICATION
+
+from tests.common import MockConfigEntry
+
+
+def _assert_result_success(result: Any) -> None:
+    """Assert successful end of flow result, creating an entry."""
+    assert result["type"] == FlowResultType.CREATE_ENTRY
```

Comment:
```suggestion
    assert result["type"] is FlowResultType.CREATE_ENTRY
```

Reply:
Also do this for the rest

Reply:
> Also do this for the rest

3 updated, other 2 were already using `is`